### PR TITLE
[DOCs/operators]: Introduction of the new RELEASE CYCLE & SANDBOX FAUCET

### PR DESCRIPTION
**Pull request by serinko**
## Problem

Nym operators community grows in quality and quantity. With node operators and developers joining the efforts to build Mixnet more robust and scalable, testing new features, sharing integration pull requests and generally taking an active part in Nym development, more transparency on the release cycle is required.

## Solution

A page clearly explaining release cycle and sandbox: *what*, *why* and *how*..

- [x] Release cycle
- [x] Ownership
- [x] Changelog and tracking
- [x] Sandbox Setup
- [x] Sandbox token faucet

- [x] Include feedback

### CODE REVIEW

File: documentation/operators/src/release-cycle.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+- *sandbox*: Public testnet, including testnet NYM token available in the [faucet](#sandbox-token-faucet)
+- *mainnet*: Nym Mixnet - the production version of Nym network
+
+## Release Flow
+
+Frequency of releases to mainnet is aimed to be every ~14 days. This time time window is an optimal compromise between periodicity and qualty assurance/testing, key factors playing an essential role in the development.
+
+|**Action** | **Environment** | **Branch** | **Ownership** |
+| :-- | :-- | :--: | :--: |
+| features development -> meged to develop | local/canary | feature branches | devs |
```
Comment (benedettadavico) [inline documentation/operators/src/release-cycle.md]: Might be easier to understand to have like:
| development work | local/canary | feature branches | devs |
| cut and test release | canary | release branch | QA |
| bug fixing | canary | directly on release branch | QA & devs |
| put release on sandbox | sandbox | release -> master/develop | QA |
| promote release to mainnet after 3-5 days | mainnet | master | QA |

possibly?

Comment (serinko) [reply documentation/operators/src/release-cycle.md]: For sure, I will redo it tomorrow!

File: documentation/operators/src/sandbox.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Create Sandbox environment config file by saving [this](https://raw.githubusercontent.com/nymtech/nym/develop/envs/sandbox.env) as `sandbox.env` in the same directory as your binaries:
+```sh
+curl -o sandbox.env -L https://raw.githubusercontent.com/nymtech/nym/develop/envs/sandbox.env
+```
+
+2. Run your `nym-node` with all the commands as always with an additional flag `--config-file` with a path to `sanbox.env` file. For example:
+```sh
+# this example is for mixnode mode
+./nym-node run --mode mixnode --config-file <PATH/TO/sandbox.env>
```
Comment (pwnfoo) [inline documentation/operators/src/sandbox.md]: I think this `--config-file` is the path to the nym-node configuration. 

It should instead be
`./nym-node --config-env-file sandbox.env  run --mode mixnode` 
or simply 
`./nym-node -c sandbox.env  run --mode mixnode`

File: documentation/operators/src/sandbox.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+# this example is for exit-gateway mode
+./nym-node run --id <ID> --config-file <PATH/TO/sandbox.env> --mode exit-gateway --public-ips "$(curl -4 https://ifconfig.me)" --hostname "<YOUR_DOMAIN>" --http-bind-address 0.0.0.0:8080 --mixnet-bind-address 0.0.0.0:1789 true --location <COUNTRY_FULL_NAME>
```
Comment (pwnfoo) [inline documentation/operators/src/sandbox.md]: same here

File: documentation/operators/src/sandbox.md (hunk 2 above)
Comment (serinko) [reply documentation/operators/src/sandbox.md]: wtf, of course you are right!

File: documentation/operators/src/sandbox.md (hunk 3 above)
Comment (serinko) [reply documentation/operators/src/sandbox.md]: +++

**Review by mfahampshire — APPROVED** [review]

**Pull request merged**